manifests/init.pp

```diff
@@ -20,6 +20,22 @@
 #
 # $pulp_max_speed::                     The maximum download speed per second for a Pulp task, such as a sync. (e.g. "4 Kb" (Uses SI KB), 4MB, or 1GB" )
 #
+# $pulp_max_tasks_per_child::           Number of tasks after which the worker is restarted and the memory it allocated is returned to the system
+#
+# $pulp_num_workers::                   Number of Pulp workers to use.
+#
+# $pulp_proxy_port::                    Port of the http proxy server
+#
+# $pulp_proxy_url::                     URL of the http proxy server
+#
+# $pulp_proxy_username::                Proxy username for authentication
+#
+# $pulp_proxy_password::                Proxy password for authentication
+#
+# $pulp_puppet_wsgi_processes::         Number of WSGI processes to spawn for the puppet webapp
+#
+# $pulp_ca_cert::                       CA certificate for pulp
```

Comment:
I feel this is a bit brief. The puppet-pulp description is: Absolute path to PEM encoded CA certificate file, used by Pulp to validate the identity of the broker using SSL.